client/pages/project-details/details.template.html.slim

```diff
@@ -2,8 +2,13 @@
   .page-header
     h1
       em Loading project data...
-.container-fluid ng-if='project && currentOrganization'
+.container-fluid.project-details ng-if='project && currentOrganization'
   .page-header
+    .btn-group.pull-right role='group' aria-label='Project Controls'
+      a.btn.btn-primary ng-if='project.repoUrl' href='{{ project.repoUrl }}' tooltip='Open repository' tooltip-append-to-body='true' target='_blank'
+        project-repo-icon project='project'
+      button.btn.btn-warning tooltip='Edit' tooltip-append-to-body='true' ui-sref='org.projects.show.edit({ projectName: project.name, id: project.id })' ng-class='{active: currentState == "org.projects.show.edit"}'
```

Comment:
You can achieve the same as `ng-class='{active: currentState == "org.projects.show.edit"}'` on this button with `ui-sref-active=true`.